Makefile

```diff
@@ -26,6 +26,10 @@ INSTALL_SIZE ?= $(shell du out/minikube-windows-amd64.exe | cut -f1)
 BUILDROOT_BRANCH ?= 2018.05.3
 REGISTRY?=gcr.io/k8s-minikube
 
+# Get git commit id
+COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
+COMMIT ?= $(if $(shell git status --porcelain --untracked-files=no),"${COMMIT_NO}-dirty","${COMMIT_NO}")
+
```

Comment:
I personally rather -dev than -dirty

Reply:
The terms have different meaning. The `-dev` means that it has a couple of commits more than the latest official release (git tag). The `-dirty` means that someone has edit the files locally, compared to the commit. That is, the binary does not (fully) reflect what is in the commit. We should make sure that official builds can be built "clean", or otherwise this will not look good.